plugins/modules/system_access_users.py

```diff
@@ -0,0 +1,251 @@
+            - The username of the OPNsense user.
+        required: true
+        type: str
+    password:
+        description:
+            - The password of the OPNsense user.
+        required: true
+        type: str
+    disabled:
+        description:
+            - Indicates whether the user account should be disabled.
+        required: false
+        default: false
+        type: bool
+    full_name:
+        description:
+            - The full name of the OPNsense user.
+        required: false
+        type: str
+    email:
+        description:
+            - The email address of the OPNsense user.
+        required: false
+        type: str
+    comment:
+        description:
+            - Additional comments or notes for the OPNsense user.
+        required: false
+        type: str
+    landing_page:
+        description:
+            - The landing page for the OPNsense user.
+        required: false
+        type: str
+    shell:
+        description:
+            - The shell for the OPNsense user.
+        required: false
+        type: str
+    expires:
+        description:
+            - The expiration date for the OPNsense user account.
+        required: false
+        type: str
+    groups:
+        description:
+            - A list of groups the OPNsense user belongs to.
+        required: false
+        type: list
+        elements: str
+    apikeys:
+        description:
+            - A list of apikeys for an OPNsense User. Generates new apikey if "" is provided.
+        required: false
+        type: list
+        elements: str
+    otp_seed:
+        description:
+            - The otp_seed of a OPNsense user.
+        required: false
+        type: str
+    authorizedkeys:
+        description:
+            - The authorizedkeys of a OPNsense user.
+        required: false
+        type: str
+    scope:
+        description:
+            - The scope of the OPNsense user.
+        required: false
+        type: str
+    uid:
+        description:
+            - The UID of the OPNsense user.
+        required: false
+        type: str
+    state:
+        description:
+            - The desired state of the OPNsense user.
+        required: false
+        choices:
+            - present
+            - absent
+        default: present
+        type: str
+'''
+
+EXAMPLES = r'''
+- name: Add OPNsense user
+  puzzle.opnsense.system_access_users:
+    username: johndoe
+    password: secret
+    full_name: John Doe
+    email: johndoe@example.com
+    groups:
+      - admins
+    state: present
+  register: result
+
+- name: Remove OPNsense user
+  puzzle.opnsense.system_access_users:
+    username: johndoe
+    state: absent
+  register: result
+'''
+
+RETURN = '''
+opnsense_configure_output:
+    description: A List of the executed OPNsense configure function along with their respective stdout, stderr and rc
+    returned: always
+    type: list
+    sample:
+      - function: "system_cron_configure"
```

Comment:
This is probably the output of the `firewall_rules` module and not the one from system_access_users

Reply:
Thank you @rekup! Fixed in 8adaf1b.